modules/appeals_api/app/services/appeals_api/pdf_construction/supplemental_claim/v4/form_data.rb

```diff
@@ -28,6 +28,25 @@ class FormData
             'fiduciary' => 5
           }.freeze
 
+          HOMELESS_LIVING_SITUATION = [
+            'I live or sleep in a place that is not meant for regular sleeping',
+            'I live in a shelter',
+            'I am staying with a friend or family member, because I am unable to own a home right now',
+            'In the next 30 days, I will have to leave a facility, like a homeless shelter',
+            'In the next 30 days, I will lose my home',
+            'None of these situations apply to me',
+            'Other'
+          ].freeze
+
+          TREATMENT_LOCATIONS = [
+            'Private health care provider',
+            'Va vet center',
+            'Community care',
+            'Va medical centers (VAMC) and community-based outpatient clinics (CBOC)',
+            'Department of defense (DOD) militart treatment facility(IES) (MTF)',
```

Comment:
Does the capitalization on these treatment locations match the form itself? I would expect to see "VA" and "Vet" and "Department of Defense," for example.

"militart" also looks like a misspelling, should probably be "military"?

Reply:
i had already asked MH, he didn't seem to really care(not in a bad way), but your suggestions are good
 and i will do

Reply:
OK, just ran it by MH again, he wants them to match the form, all caps they are